src/figuresClasses.ts

```diff
@@ -1,19 +1,97 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: 'triangle';
+
+  color: 'red' | 'green' | 'blue';
```

Comment:
Use types. Check other similar cases.

Reply:
also instead thix line you can use public keyword in constructor